project.mml

```diff
@@ -1475,6 +1475,7 @@ Layer:
                       WHEN (tags ? 'ele') AND tags->'ele' ~ '^-?\d{1,4}(\.\d+)?$'
                         AND ("natural" IN ('peak', 'volcano', 'saddle')
                           OR tourism = 'alpine_hut' OR (tourism = 'information' AND tags->'information' = 'guidepost')
+                          OR tourism = 'wilderness_hut'
```

Comment:
Combine this into the alpine_hut condition by modifying it to `tourism IN ('alpine_hut', 'wilderness_hut')`

Reply:
Thanks, fixed.